fairseq/options.py

```diff
@@ -591,6 +591,10 @@ def add_generation_args(parser):
                        help='if set, the last checkpoint are assumed to be a reranker to rescore the translations'),
     group.add_argument('--retain-iter-history', action='store_true',
                        help='if set, decoding returns the whole history of iterative refinement')
+    group.add_argument('--retain-dropout', action='store_true',
+                       help='Use dropout at inference time')
+    group.add_argument('--retain-dropout-modules', default=None, nargs='+', type=str,
+                       help='Use dropout at inference time only for specific modules.')
```

Comment:
Yes, as it is defined now, it is applied to all modules by default. But the user might want to avoid applying it to some specific modules. E.g. we didn't apply it to the attention modules in our experiments. Maybe it would make more sense if I change the option `--retain-dropout-modules` to `--exclude-dropout-modules` or something like that?